HOWTO-make-a-release.md

```diff
@@ -0,0 +1,377 @@
+Just copy and paste existing announcements making minor changes for the date
+and version number as necessary.  If there is an advisory then ensure you
+include a link to it.
+
+Update the news/vulnerabilities.xml file if appropriate.
+
+If there is a Security Advisory then copy it into the news/secadv directory.
+
+Commit your changes, but *do not push* them to the website repo at this stage.
+(the public repo being `openssl-git@git.openssl.org:openssl-web.git`)
+
+*Do* send the commits to the reviewer and await their approval.
+
+# Publish the release
+
+*BE CAREFUL*  This section makes everything visible and is therefore largely
+irreversible.  If you are performing a dry run then DO NOT perform any steps
+in this section.
+
+Check that the release has been uploaded properly.  The release tarballs and
+associated files should be in ~openssl/dist/new.  They should be owned by
+the openssl userid and world-readable.
+
+Copy the tarballs to appropriate directories.  This can be done using the
+do-release.pl script.  See $TOOLS/release-tools/DO-RELEASE.md for a
+description of the options.  For example:
+
+    sudo -u openssl perl ~openssl/do-release.pl --copy --move
+
+This will copy the relevant files to the website and move them from
+`~openssl/dist/new` to `~openssl/dist/old` so they will not seen by a
+subsequent release.  Alternatively if you want to perform one release at a
+time or copy/move the files manually, see below.
+
+The do-release.pl script will display the commands you will need to issue to
+send the announcement emails later.  Keep a note of those commands for
+future reference.
+
+Verify that the tarballs are available via FTP:
+
+        ftp://ftp.openssl.org/source/
+
+And that they are ready for the website:
+
+        ls /var/www/openssl/source
+
+*For OpenSSL 3.0 and on*, push your local changes to the public source repo
+as instructed by `dev/release.sh`.  You may want to sanity check the pushes
+by inserting the `-n` (dry-run) option.
+
+*For OpenSSL before 3.0*, simply push your local changes to the public
+source repo, and please do remember to push the release tags as well, which
+is done separately with the `--tags` option.  You may want to sanity check
+the pushes by inserting the `-n` (dry-run) option.
+
+## Updating the website
+
+Push the website changes you made earlier to the OpenSSL website repo.  When
+you do this, the website will get updated and a script to flush the Akamai CDN
+cache will be run.  You can look at things on www-origin.openssl.org; the
+CDN-hosted www.openssl.org should only be a few minutes delayed.
+
+# Post-publishing tasks
+
+## Check the website
+
+Verify that the release notes, which are built from the CHANGES file in the
+release, have been updated.  This is done automatically by the commit-hook,
+but if you see a problem, try the following steps on `dev.openssl.org`:
+
+    cd /var/www/openssl
+    sudo -u openssl -H make relupd
+    sudo -u openssl ./bin/purge-one-hour
```

Comment:
This doesn't work. It needs a "-H" after openssl.

Reply:
Okie